index.html

```diff
@@ -44,5 +43,140 @@ <h3 class="small-txt">LET'S CONNECT</h3>
 
    </header> 
```

Comment:
Kindly, by convention the HEADER tag is used to contain the top bar and within it find the NAVBAR, I recommend you make this change so that your site is better structured.